src/PayPlugin.js

```diff
@@ -27,7 +27,7 @@ export default class PayPlugin extends FlexPlugin {
         //loadCSS("http://localhost:3000/styles.css");
 
         const options = { sortOrder: -1 };
-        flex.AgentDesktopView.Panel2.Content.add(
+        flex.AgentDesktopView.Panel2.Content.replace(
```

Comment:
The reason add is used here rather than replace is so that the plugin can be used alongside other Panel2 plugins. If you need to remove the rest of the Panel2 it would be better either do that separately to adding the Payment panel.